### ISSUE TITLE
Still default theme after installing Material Theme

### ISSUE THREAD

**Issue by dev1337** [equinusocio/vsc-material-theme#84]
I'm running Visual Studio Code 1.15.1 on macOS and I've followed the installation instructions as closely as possible. Certain steps seem to be outdated, specifically `ext install vsc-material-theme`. But the theme does not seem to work.

I still see the Visual Studio Code default theme despite having activated it according to the last installation steps by going to the Material Theme settings and picking a color variant.

**Comment by equinusocio** [comment]
@OctoD can you check this?

**Comment by OctoD** [comment]
@equinusocio got it, i'll try to replicate this behaviour.

**Comment by OctoD** [comment]
I'm trying to figure it out but I cannot replicate this behaviour; 

activating through these steps it works

1 - install the theme (of course you did! :-) )
2 - choose Material Theme among the others themes
3 - choose Material Icon Theme (recommended)
4 - set you variant and/or accents
 
did you do the same thing?

**Comment by dev1337** [comment]
It's my first time on macOS and VSCode and it took me a while to figure out what I was doing wrong. 

It appears that I was actually following all the steps correctly without actually having selected the Material Theme in the color theme preferences. Instead I went ahead and picked a color variant directly in the Material Settings thinking it was supposed to change my current theme in my preferences.

Silly me, but I got it to work! 😁
